components/lightbox/template-preview.snip.html

```diff
@@ -1,97 +0,0 @@
-{{!
```

Comment:
I assume you delete the "components" directory on purpose (they are not related to the templates)

Reply:
yes, we can re-add it later